boards/arm/96b_neonkey/Kconfig.defconfig

```diff
@@ -10,4 +10,21 @@ if BOARD_96B_NEONKEY
 config BOARD
 	default 96b_neonkey
 
+if I2C
+
+config I2C_1
+	default y
+
+endif # I2C
+
+if SPI
+
+config SPI_1
+	default y
+
+config SPI_1_IRQ_PRI
```

Comment:
Same here, and elsewhere -- IRQ priorities are up to the application, not the board, no? There's nothing board-specific about this I can see.

Reply:
Same as above.